README.rst

```diff
@@ -267,3 +281,4 @@ When using the AutoExtract providers, be aware that:
 .. _`Scrapy's asyncio documentation`: https://docs.scrapy.org/en/latest/topics/asyncio.html
 .. _`Request-level error`: https://doc.scrapinghub.com/autoextract.html#request-level
 .. _`Query-level error`: https://doc.scrapinghub.com/autoextract.html#query-level
+.. _`supported page types`: https://autoextract-poet.readthedocs.io/en/more_page_types/_autosummary/autoextract_poet.pages.html
```

Comment:
I think we should use either latest or stable branch in this link, no more_page_types

Reply:
Good catch. Fixed in https://github.com/scrapinghub/scrapy-autoextract/pull/25/commits/380a377953f137cf2c4f60bcd4b8c6ce61b68428. I see that `_autosummary` is also part of the URL, this is ugly. I would have chosen a different name if I would have known it. Something to be changed in the future.